### PR TITLE
dependencies upgrade

### PR DESCRIPTION
**Pull request by tutok**
### Please check if the PR fulfills these requirements ###

- [x] The commit message follows our guidelines

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
upgraded/aligned versions of dependencies:
- `@babel/runtime`
- `@types/react`
- `@types/react-dom`
- `apollo-client`
- `enzyme-adapter-react-16`
- `prop-types`
- `react-dom`
- `react-apollo`
- `react-dom`
- `react-test-renderer`

### CODE REVIEW

File: packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx [hunk 1]
```diff
@@ -51,3 +52,8 @@ export const DynamicRoute: React.SFC<DynamicRouteProps> = props => {
     </Router>
   );
 };
+DynamicRoute.propTypes = {
+  location: PropTypes.any,
+  components: PropTypes.any.isRequired,
+  notFound: PropTypes.func.isRequired
```
Comment (Borales) [inline packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx]:
```suggestion
  notFound: PropTypes.element.isRequired
```
?

File: packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx [hunk 2]
```diff
@@ -51,3 +52,8 @@ export const DynamicRoute: React.SFC<DynamicRouteProps> = props => {
     </Router>
   );
 };
+DynamicRoute.propTypes = {
+  location: PropTypes.any,
+  components: PropTypes.any.isRequired,
```
Comment (Borales) [inline packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx]:
```suggestion
  components: PropTypes.objectOf(PropTypes.element).isRequired,
```
not sure about this one

File: packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx [hunk 3]
```diff
@@ -51,3 +52,8 @@ export const DynamicRoute: React.SFC<DynamicRouteProps> = props => {
     </Router>
   );
 };
+DynamicRoute.propTypes = {
+  location: PropTypes.any,
```
Comment (Borales) [inline packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx]:
```suggestion
  location: PropTypes.string,
```

File: packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx (hunk 1 above)
Comment (tutok) [reply packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx]: it produces followign error:
```
Type 'Validator<ReactElementLike>' is not assignable to type 'Validator<ComponentType<any>>'
```
I gave up with solving it. if you have any idea how to deal with it please share ;)

File: packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx (hunk 2 above)
Comment (tutok) [reply packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx]: this is ok
```suggestion
components: PropTypes.objectOf(PropTypes.func.isRequired).isRequired,
```

File: packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx (hunk 3 above)
Comment (tutok) [reply packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx]: actually, `location` is a `shape`, but still had some typing issue.
```export type ValidationMap<T> = { [K in keyof T]?: Validator<T[K]> };```
it expect some additional proptype alidator for indexer which is wrong

File: packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { Redirect } from 'react-router-dom';
+import PropTypes, { ReactNodeLike } from 'prop-types';
```
Comment (mateuszwozniak) [inline packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx]:
```suggestion
import PropTypes from 'prop-types';
```

Comment (tutok) [reply packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx]: thanks!

**Pull request merged**